lib/milc_interface.cpp

```diff
@@ -61,19 +66,20 @@ using namespace quda::fermion_force;
 
 
 #define QUDAMILC_VERBOSE 1
+
 template <bool start> void inline qudamilc_called(const char *func, QudaVerbosity verb)
 {
+  PUSH_RANGE(func, 1);
 #ifdef QUDAMILC_VERBOSE
-if (verb >= QUDA_VERBOSE) {
-  if (start) {
-    printfQuda("QUDA_MILC_INTERFACE: %s (called) \n", func);
-    PUSH_RANGE(func, 1);
-  } else {
-    printfQuda("QUDA_MILC_INTERFACE: %s (return) \n", func);
-    POP_RANGE;
+  if (verb >= QUDA_VERBOSE) {
+    if (start) {
+      printfQuda("QUDA_MILC_INTERFACE: %s (called) \n", func);
+    } else {
+      printfQuda("QUDA_MILC_INTERFACE: %s (return) \n", func);
```

Comment:
this looks like it breaks the behaviour of the interface mark up so that a interface routine

Reply:
No, let me fix that ...